images-list

```diff
@@ -51,6 +57,9 @@ prom/prometheus rancher/prom-prometheus v2.18.2
 quay.io/kiali/kiali rancher/kiali-kiali v1.22.1
 quay.io/kiali/kiali rancher/kiali-kiali v1.23.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.24.0
+quay.io/prometheus-operator/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.43.0
+quay.io/prometheus-operator/prometheus-operator rancher/coreos-prometheus-operator v0.43.0
+quay.io/prometheus/prometheus rancher/prom-prometheus v2.22.0
```

Comment:
@cbron as mentioned in the issue description, the following already have Rancher repos but aren't in the scripts:
- https://hub.docker.com/r/rancher/coreos-prometheus-config-reloader/tags
- https://hub.docker.com/r/rancher/coreos-prometheus-operator/tags
- https://hub.docker.com/r/rancher/prom-prometheus/tags

Do we need an EIO ticket to track enabling the repositories?

Reply:
They shouldn't, but we can ask EIO to double check that they have drone -> docker credentials set correctly.

Reply:
They verified the permissions, so it should be good to go.